plugins/aladino/functions/hasRequiredApprovals.go

```diff
@@ -0,0 +1,73 @@
+package plugins_aladino_functions
+
+import (
+	"fmt"
+
+	"github.com/reviewpad/reviewpad/v3/codehost/github/target"
+	"github.com/reviewpad/reviewpad/v3/handler"
+	"github.com/reviewpad/reviewpad/v3/lang/aladino"
+)
+
+func HasRequiredApprovals() *aladino.BuiltInFunction {
+	return &aladino.BuiltInFunction{
+		Type:           aladino.BuildFunctionType([]aladino.Type{aladino.BuildIntType(), aladino.BuildArrayOfType(aladino.BuildStringType())}, aladino.BuildBoolType()),
```

Comment:
Suggestion for better reading
```suggestion
		Type: aladino.BuildFunctionType(
			[]aladino.Type{
				aladino.BuildIntType(),
				aladino.BuildArrayOfType(aladino.BuildStringType())
			},
			aladino.BuildBoolType(),
		),
```